packages/sync/src/links/OfflineQueueLink.ts

```diff
@@ -62,12 +63,27 @@ export class OfflineQueueLink extends ApolloLink {
     this.operationFilter = filter;
   }
 
-  public open() {
+  public async open() {
     logger("MutationQueue is open", this.opQueue);
     this.isOpen = true;
-    this.opQueue.forEach(({ operation, forward, observer }) => {
-      forward(operation).subscribe(observer);
-    });
+    for (const opEntry of this.opQueue) {
+      const { operation, forward, observer } = opEntry;
+      await new Promise(resolve => {
```

Comment:
Better to extract this to separate method: `updateId's` to make this more maintainable.

Reply:
I've created `forwardQueuedOperation` method [1] which basically makes possible to await `forward`.

[1] https://github.com/aerogear/aerogear-js-sdk/pull/220/files#diff-334f3fa2eb415fbeb3824ef3b38b5580R115